code/data_structures/src/stack/prefix_to_postfix/prefix_to_postfix.cpp

```diff
@@ -0,0 +1,61 @@
+#include <iostream> 
+#include <stack> 
+using namespace std; 
```

Comment:
Don't use this. Reformat this file using clang format 4 spaces.

Reply:
Ok, sir, I will do this.

Reply:
@arnavb sir, will u please give me an example of this format?

Reply:
@rudrakshi99 Try [this website](https://zed0.co.uk/clang-format-configurator/). Set the indent width to 4 spaces.